### PR TITLE
Updated CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Vani177**
#21 

The changes in this PR include the steps required to locally develop and test the website.

For the reference of before and after the changes I've made in the file:
image

I hope this serves the purpose. I'd be glad to make any changes you suggest :)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+Please make sure you have read the following criteria before making a contribution.
+<br>
+
+**1.  Fork this repository**
+<br>
+**2. Clone the fork to your local machine using the git clone command**
+<br>
+   ```
+   github clone
+   https://github.com/yourusername/repository-name.git
```
Comment (AdarshRawat1) [inline CONTRIBUTING.md:14]: Add the link to this repository.
Reason - This is a guide for this repository

Comment (Vani177) [reply CONTRIBUTING.md:14]: Sure, noted

**Comment by AdarshRawat1** [comment]
Please provide images that show the step-by-step process. This will simplify the understanding for those contributing for the first time.

**Comment by Vani177** [comment]
Oh sure, will do!

**Comment by Vani177** [comment]
hey! I've updated the file with new changes, is it alright now?

**Comment by AdarshRawat1** [comment]
> The changes in this PR include the steps required to locally develop and test the website.

Looks great!! 
> For the reference of before and after the changes I've made in the file: 

I have a suggestion - You should consider including a screenshot of the output file (README.md).
Reason -  changes to code are visible in  `Files Changed` section to maintainer. 
image

> I hope this serves the purpose. I'd be glad to make any changes you suggest :)

Thank you for the PR.

**Pull request merged**